src/style.css

```diff
@@ -0,0 +1,179 @@
+:root {
+  --primary: #fc466b;
+  --light: #eee;
+  --lighter: #fff;
+  --dark: #313154;
+  --darker: #000;
+  --danger: #c9d3fd;
+  --grey: #888;
+  --shadow: 0 1px 3px rgba(211, 0, 0, 0.027);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+nav {
+  background-color: var(--dark);
+  font-size: 2vw;
+  display: flex;
+  justify-content: space-between;
+  padding-inline: 5%;
+}
+
+.leftHeader,
+ul {
+  display: flex;
```

Comment:
- [ ] `display: flex` re-occurs to many times in this file, creating a reusable class and reusing it where needed would help you achieve [HTML and CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly do.